tests/attacks/test_hop_skip_jump.py

```diff
@@ -81,6 +81,24 @@ def test_tensorflow_mnist(self):
         y_pred_adv = np.argmax(tfc.predict(x_test_adv), axis=1)
         self.assertTrue((target == y_pred_adv).any())
 
+        # Test the masking 1
```

Comment:
I think the tests check if the unmasked input does not get modified, which is good. Could you please also add a test to check if the masked input gets perturbed.

Reply:
Solved.